source/client.c

```diff
@@ -798,6 +859,8 @@ static void s_subscribe_complete(
 
     struct subscribe_task_arg *task_arg = userdata;
 
+    AWS_LOGF_DEBUG(AWS_LS_MQTT_CLIENT, "Subscribe %d completed", packet_id);
```

Comment:
This seems like an int size mismatch but somehow the gcc compiler check isn't complaining?

Reply:
Also, maybe log error vs. no-error state (several spots in all these _complete callbacks)

Reply:
Can fix, I'll use `PRIu16`